library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -215,6 +215,20 @@ button[class="disabled"]:hover,
 	text-decoration: none;
 }
 
+input[type="checkbox"].formButtonDisabled{
+	@include border-radius(3px);
+	&:checked{
+		background-image: url("images/check.png") !important;
```

Comment:
yeah you would need to untangle the other !important as well, which is typically solved by css scoping.